optaplanner-examples/src/main/java/org/optaplanner/examples/curriculumcourse/domain/Curriculum.java

```diff
@@ -25,6 +27,15 @@
 
     private String code;
 
+    public Curriculum() {
+
```

Comment:
soft codes tyle: typically we don't leave that empty line there in an empty constructor.